lib/ex_doc.ex

```diff
@@ -37,16 +38,17 @@ defmodule ExDoc do
        deps: [{ebin_path :: String.t, doc_url :: String.t}],
```

Comment:
@josevalim according to what you said, I am uncertain whether these typespecs should be `String.t` or `nil | String.t`